otter/integration/lib/mimic.py

```diff
@@ -0,0 +1,36 @@
+"""Contains reusable classes relating to mimic."""
+
+from __future__ import print_function
+
+import json
+
+from characteristic import Attribute, attributes
+
+import treq
+
+from otter.util.http import check_success, headers
+
+
+@attributes(["pool", Attribute("treq", default_value=treq)])
+class MimicNova(object):
+    """
+    Class that handles HTTP requests to the mimic nova control plane.
+    """
+    def change_server_statuses(self, rcs, ids_to_status):
+        """
+        Change the statuses of the given server IDs.
+
+        :param rcs: A :class:`otter.integration.lib.resources.TestResources`
+            instance.
+        :param dict ids_to_status: A mapping of server IDs to the string status
```

Comment:
Non-blocking: Should we include a list of the possible status values here, or just reference the documentation?